### ISSUE TITLE
为啥多标签模型训练速度这么慢 

### ISSUE THREAD

**Issue by wangzepeng111** [PaddlePaddle/Paddle#42048]
### 请提出你的问题 Please ask your question

keras代码
from keras.optimizers import adam_v2
def deep_model(feature_dim,label_dim):
    from keras.models import Sequential
    from keras.layers import Dense
    model = Sequential()
    print("create model. feature_dim ={}, label_dim ={}".format(feature_dim, label_dim))
    model.add(Dense(4, activation='relu', input_dim=feature_dim))
    model.add(Dense(16, activation='relu'))
    model.add(Dense(128, activation='relu'))
    model.add(Dense(1024, activation='relu'))
    model.add(Dense(128, activation='relu'))
    model.add(Dense(label_dim, activation='sigmoid'))
    model.compile(optimizer=adam_v2.Adam(lr = 1e-4), loss='binary_crossentropy', metrics=[f1])
    return model

paddle代码
class MNIST(paddle.nn.Layer):
    def __init__(self):
        super(MNIST, self).__init__()
        # 定义两层全连接隐含层，输出维度是10，当前设定隐含节点数为10，可根据任务调整
        self.fc1 = Linear(in_features=1, out_features=128)
        self.fc2 = Linear(in_features=128, out_features=1024)
        # 定义一层全连接输出层，输出维度是1
        self.fc3 = Linear(in_features=1024, out_features=2884)
        self.fc4 = Linear(in_features=2884, out_features=512)
        self.fc5 = Linear(in_features=512, out_features=128)
        self.fc6 = Linear(in_features=128, out_features=15)
        self.dropout = paddle.nn.Dropout(0.2)
    # 定义网络的前向计算，隐含层激活函数为sigmoid，输出层不使用激活函数
    @paddle.jit.to_static
    def forward(self, inputs):
        # inputs = paddle.reshape(inputs, [inputs.shape[0], 784])
        outputs1 = self.fc1(inputs)
        outputs1 = F.relu(outputs1)
        outputs1= self.dropout(outputs1)
        outputs2 = self.fc2(outputs1)
        outputs2 = F.relu(outputs2)
        outputs2= self.dropout(outputs2)
        outputs3 = self.fc3(outputs2)
        outputs3 = F.relu(outputs3)
        outputs3= self.dropout(outputs3)
        outputs4 = self.fc4(outputs3)
        outputs4 = F.relu(outputs4)
        outputs4= self.dropout(outputs4)
        outputs5 = self.fc5(outputs4)
        outputs5 = F.relu(outputs5)
        outputs_final = self.fc6(outputs5)
        return outputs_final
paddle至少要半小时才能跑500epoch 同样的keras只需要几分钟

**Comment by iclementine** [comment]
close 了？是发现了什么原因还是已经解决了吗？

**Comment by wangzepeng111** [comment]
计算多标签的F1每次都计算耗时太久啦 我让它每个batch都算f1它算不过来 keras设计的应该就是每个epoch才计算f1评估矩阵
